src/common/logging/BUILD

```diff
@@ -1,3 +1,20 @@
+#-------------------------------------------------------------------------------
+# Copyright 2022 Google LLC
```

Comment:
Just checking, is this 2022? Logging is old in the project, but I don't know about this `BUILD` file off the top of my head.